rtlib/generic_opts.hh

```diff
@@ -99,23 +151,55 @@ class Opts {
 #ifdef LIBRNA_RNALIB_H_
         << " (-[tT] [0-9]+)? (-P PARAM-file)?"
 #endif
-        << " (-[drk] [0-9]+)* (-h)? (INPUT|-f INPUT-file)\n";
-    }
+        << " (-[drk] [0-9]+)* (-h)? (INPUT|-f INPUT-file)\n"
+        << "--help,-H,-h                          print this help message\n"
+#ifdef CHECKPOINTING_INTEGRATED
+        << "--checkpointInterval,-c    d:h:m:s    specify the checkpointing "
+        << "interval, default: 0:0:1:0 (1h)\n"
+        << "--checkpointOutput,-O      PATH       set the path where to store "
+        << "the checkpoints, default: current working directory\n"
+        << "                                      The program will also attempt"
+        << " to read existing checkpoints from a Logfile from this path.\n"
+        << "                                      Use the --checkpointInput "
```

Comment:
hm, I tend to say that the re-execution should not do this "magically", i.e. automatics. It might be more clear to the user, if she/he has to invoke this manually?! What are your takes @kmaibach @fymue ?

Reply:
I agree, this might be a little unclear to the user.
I implemented this to enable one of your earlier suggestions to warn the user if the current command line inputs differ from the command line inputs of the command that produced the to-be-read checkpoints. Since it wouldn't make much sense to just log previous command line inputs without any information regarding where the checkpoint archives they produced are saved, I figured I might as well try to automatically read from these checkpoint paths for convenience reasons since I need to log them anyways. 
But I can of course also turn off the automatic adoption of the paths from the Logfile and require the user to specify the input path while still comparing the checkpoint archives at this user-specified path with the information from the Logfile to potentially warn the user if the command line inputs differ.

Reply:
So the question is if existing checkpoints should be automatically read from the log file when recalculating unfinished jobs.

I agree with Fynn that this would be convenient, but it might lead to users not thinking about what this implies or not even knowing this happens in the first place.
Therefore I have two suggestions:

1. implement a new parameter that can enable this feature (i.e., for convenience purposes)
or:
2. disable the automatic execution altogether, as you suggested, and force the user to manually declare the path to their corresponding checkpoints